build.gradle

```diff
@@ -31,7 +31,7 @@ buildscript {
 plugins {
     id 'com.github.johnrengelman.shadow' version '5.1.0'
     id 'com.palantir.configuration-resolver' version '0.4.0'
-    id 'com.palantir.consistent-versions' version '1.17.3'
+    id 'com.palantir.consistent-versions' version '1.26.1'
```

Comment:
Note this upgrade reduced the gradle `CONFIGURING` time from ~50 seconds to ~4 seconds on my machine.